docs/topics/api/addons.rst

```diff
@@ -192,6 +180,76 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
         dictionary  Dictionary
     ==============  ==========================================================
 
+-------------
+Versions List
+-------------
+
+.. _version-list:
+
+This endpoint allows you to list all versions belonging to a specific add-on.
+
+.. http:get:: /api/v3/addons/addon/(int:addon_id|string:addon_slug|string:addon_guid)/versions/
```

Comment:
I find the choice between the 3 identifiers very useful for quickly access information about an add-on, so I dunno. The add-on detail page at least need to have that choice to allow admins to find an add-on with one of those identifiers.